web/src/stories/button.group.stories.mdx

```diff
@@ -12,19 +12,28 @@ buttonGroup.AddButton(
       )))
 `}}
```

Comment:
I think I'd prefer to see stories under `LinkComponent` show how to add a button as a link.